scripts/bin-test/sources/commonjs-main/functions.js

```diff
@@ -16,3 +18,27 @@ exports.v2http = functionsv2.https.onRequest((req, resp) => {
 exports.v2callable = functionsv2.https.onCall(() => {
     return "PASS";
 });
+
+// A Firebase extension by ref
+const extRef1 = firestoreTranslateText("extRef1", {
+    "COLLECTION_PATH": "collection1",
+    "INPUT_FIELD_NAME": "input1",
+    "LANGUAGES": "de,es",
+    "OUTPUT_FIELD_NAME": "translated",
+    "_EVENT_ARC_REGION": "us-central1",
+    "_FUNCTION_LOCATION": "us-central1",
+});
+exports.extRef1 = extRef1;
```

Comment:
nit* why not

```
exports.extRef1 = firestoreTranslateText("extRef1", {
    "COLLECTION_PATH": "collection1",
    "INPUT_FIELD_NAME": "input1",
    "LANGUAGES": "de,es",
    "OUTPUT_FIELD_NAME": "translated",
    "_EVENT_ARC_REGION": "us-central1",
    "_FUNCTION_LOCATION": "us-central1",
});
```

Reply:
Done for the ones not referenced by functions. Also removed the extensions from the parameters test.